week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,23 @@
+function percentage() {
```

Comment:
This exercise hasn't been finished, it still requires a bit of work to understand how to use functions. I recommend looking at your solution for the exercise at I-floats/exercise.js and then thinking about how to place that code in separate functions: one for calculating percentages and one for creating the message. You should also attempt exercise2: https://github.com/CodeYourFuture/js-exercises/tree/master/week-1/L-functions-nested